src/components/Header.tsx

```diff
@@ -26,8 +43,28 @@ const Header: FC = () => {
           >
             <CalculateIcon sx={{ fontSize: 32 }} />
             <Typography variant="h5" component="div" sx={{ flexGrow: 1 }}>
-              シャーレの総決算with連邦生徒会 在庫管理計算機
+              {t('typography')}
             </Typography>
+            <Container
+              sx={{
+                display: 'flex',
+                justifyContent: 'flex-end',
+                alignItems: 'center',
+                gap: 1,
+              }}
+            >
```

Comment:
After adding the Container, the title in the AppBar is now wrapping into two lines instead of staying on one line. It looks like specifying a width will fix the issue. Could you please update it accordingly?

```suggestion
            <Container
              sx={{
                display: 'flex',
                justifyContent: 'flex-end',
                alignItems: 'center',
                gap: 1,
                width: 'auto',
              }}
            >
```